manifests/3.0.0/opensearch-3.0.0.yml

```diff
@@ -23,6 +23,17 @@ components:
     checks:
       - gradle:publish
       - gradle:properties:version
+  - name: opensearch-learning-to-rank-base
+    repository: https://github.com/opensearch-project/opensearch-learning-to-rank-base
+    ref: main
+    platforms:
+      - linux
+      - windows
+    checks:
+      - gradle:publish
+      - gradle:properties:version
+    depends_on:
+      - common-utils
```

Comment:
Hi @rithin-pullela-aws  can you add this below JS or at the end of the file? So that other dependencies can be build first just in case?